ui_framework/src/global_styling/reset/_reset.scss

```diff
@@ -24,8 +30,8 @@ time, mark, audio, video {
   padding: 0;
   border: 0;
   font-size: 100%;
-  font: inherit;
-  font-family: $kuiFontFamily;
+  font: inherit; /* 1 */
+  font-family: inherit; /* 1 */
```

Comment:
I find it hard to read. You have to scan multiple lines to see the comment. I don't see this style anywhere in larger frameworks. They all use single line comments either above or next to the properties in use. 

I don't see much use in adding a key for CSS blocks. At most you're looking at under 10 defined properties. Rather just see it in use against the content where it's being defined.

It's a small place where I think their recommendation is off.